libibverbs/verbs.h

```diff
@@ -602,7 +608,14 @@ struct ibv_wc {
 	unsigned int		wc_flags;
 	uint16_t		pkey_index;
 	uint16_t		slid;
-	uint8_t			sl;
+	/* SL for IB or packet type
+	 * (GRH/IPv4/IPv6) for RoCE in the CQE
```

Comment:
@jgunthorpe ????

Reply:
Hi, Leon. Let me explain the purpose of this patch more clearly.

According to Annex17_RoCEv2 (A17.4.5.2), for RoCEv2 UD, a CQE should carry a flag that indicates if the received
frame is an IPv4, IPv6 or RoCE packet. But currently, the values of the flag corresponding to these packet types haven't been defined yet in WC. As these values have already been defined In the UCX patch of https://github.com/openucx/ucx/commit/ed28845b88633e65d64fce8ec880060aa61bd59c, I think they should also be specified in WC to support UCX.

Reply:
You can't change this struct, it is ABI.